agent/lib/kubernetes.js

```diff
@@ -85,7 +92,11 @@ function list_options(resource, baseOptions) {
 
 function watch_options(resource, baseOptions) {
     var base = resource.startsWith("address") ? '/apis/enmasse.io/v1beta1/watch/namespaces/' : '/api/v1/watch/namespaces/';
-    var options = get_options(baseOptions, get_path(base, resource, baseOptions));
+    var options = get_options(baseOptions, get_path(base, resource, includeResyncInterval(baseOptions)));
```

Comment:
I find applying resync interval twice distasteful and hard to follow.
WDYT about: https://github.com/vbusch/enmasse/pull/6/

Reply:
getOptions won't preserve the requestTimeout.
Does my latest commit help?  https://github.com/EnMasseProject/enmasse/pull/4137/commits/c056a1018cf6fafa62e83a9d02a13f2ef565d345

Reply:
Yes, it does.   thanks you.